pandas/core/missing.py

```diff
@@ -223,13 +223,22 @@ def _interpolate_scipy_wrapper(x, y, new_x, method, fill_value=None,
         raise ImportError('{0} interpolation requires Scipy'.format(method))
 
     new_x = np.asarray(new_x)
-
-    # ignores some kwargs that could be passed along.
-    alt_methods = {
-        'barycentric': interpolate.barycentric_interpolate,
-        'krogh': interpolate.krogh_interpolate,
-        'piecewise_polynomial': interpolate.piecewise_polynomial_interpolate,
-    }
+    import scipy
+    if scipy.__version__ <= '0.17':
```

Comment:
Could you move this up to be with the `from scipy import interpolate` and assign a variable like `scipy_version = scipy.__version__`. Then your if block would me `if LooseVersion(scipy_version) <= LooseVersion('0.17'):`, where `LooseVersion` comese from `distutils.version`.